Translations/pt_BR.po

```diff
@@ -4751,481 +4729,331 @@ msgid "Litch Demon"
 msgctxt "monster"
 msgid "Hollow One"
-msgstr "espada de uma mão"
+msgstr "Oco"
 
 #: Source/monstdat.cpp:119
-#, fuzzy
-#| msgid "Pain Master"
 msgctxt "monster"
 msgid "Pain Master"
-msgstr "da dor"
+msgstr "Mestre da Dor"
 
 #: Source/monstdat.cpp:120
-#, fuzzy
-#| msgid "Reality Weaver"
 msgctxt "monster"
 msgid "Reality Weaver"
 msgstr "Tecelão de Realidades"
 
 #: Source/monstdat.cpp:121
-#, fuzzy
-#| msgid "Succubus"
 msgctxt "monster"
 msgid "Succubus"
 msgstr "Súcubo"
 
 #: Source/monstdat.cpp:122
-#, fuzzy
-#| msgid "Snow Witch"
 msgctxt "monster"
 msgid "Snow Witch"
-msgstr "Cabana da bruxa"
+msgstr "Bruxa da Neve"
 
 #: Source/monstdat.cpp:123
-#, fuzzy
-#| msgid "Hell Spawn"
 msgctxt "monster"
 msgid "Hell Spawn"
-msgstr "Descer para o inferno"
+msgstr "Cria do Inferno"
 
 #: Source/monstdat.cpp:124
-#, fuzzy
-#| msgid "Soul Burner"
 msgctxt "monster"
 msgid "Soul Burner"
 msgstr "Queimadora de Almas"
 
 #: Source/monstdat.cpp:125
-#, fuzzy
-#| msgid "Counselor"
 msgctxt "monster"
 msgid "Counselor"
 msgstr "Conselheiro"
 
 #: Source/monstdat.cpp:126
-#, fuzzy
-#| msgid "Magistrate"
 msgctxt "monster"
 msgid "Magistrate"
 msgstr "Magistrado"
 
 #: Source/monstdat.cpp:127
-#, fuzzy
-#| msgid "Cabalist"
 msgctxt "monster"
 msgid "Cabalist"
 msgstr "Cabalista"
 
 #: Source/monstdat.cpp:128
-#, fuzzy
-#| msgid "Advocate"
 msgctxt "monster"
 msgid "Advocate"
 msgstr "Defensor"
 
 #: Source/monstdat.cpp:129
-#, fuzzy
-#| msgid "Golem"
 msgctxt "monster"
 msgid "Golem"
-msgstr "Pergaminho de golem"
+msgstr "Golem"
 
 #: Source/monstdat.cpp:130
-#, fuzzy
-#| msgid "The Dark Lord"
 msgctxt "monster"
 msgid "The Dark Lord"
-msgstr ""
-"Então a lenda do Mapa é real. Até eu nunca acreditei em nada sobre ele! "
-"Imagino que chegou a hora de lhe dizer a verdade sobre quem eu sou, "
-"meu(inha) amigo(a). Eu não sou o que pareço ser...\n"
-" \n"
-"Meu verdadeiro nome é Deckard Cain, o Ancião, e eu sou o último descendente "
-"de uma antiga Irmandade que era dedicada a salvaguardar os segredos de um "
-"mal atemporal. Um mal que muito obviamente foi agora liberto...\n"
-" \n"
-"O mal contra o qual você se move é o sombrio Senhor do Terror. Ele é "
-"conhecido entre os homens mortais como Diablo. Ele é quem foi aprisionado "
-"dentro do Labirinto há muitos séculos. O Mapa que você agora possui foi "
-"criado eras atrás para marcar o momento em que Diablo se levantaria "
-"novamente de sua prisão. Quando as duas estrelas nesse mapa se alinharem, "
-"Diablo estará no auge de seu poder. Ele será simplesmente invencível...\n"
-" \n"
-"Você está agora em uma corrida contra o tempo, meu(inha) amigo(a)! Encontre "
-"Diablo antes que as estrelas se alinhem, pois talvez nunca mais tenhamos "
-"outra chance de livrar o mundo de seu mal!"
+msgstr "O Lorde Sombrio"
 
 #: Source/monstdat.cpp:131
-#, fuzzy
-#| msgid "The Arch-Litch Malignus"
 msgctxt "monster"
 msgid "The Arch-Litch Malignus"
-msgstr "de arcanjo"
+msgstr "The Arch-Litch Malignus"
```

Comment:
This one _should_ have a translation.... "The" is missing,  "arch" is "arce", but I don't know if "lich"/"litch" exists...

Maybe we should just go with:
```suggestion
msgstr "O Arce-Litch Malignus"
```
?